spring-kafka/src/test/java/org/springframework/kafka/retrytopic/RetryTopicConfigurerTests.java

```diff
@@ -362,14 +364,23 @@ void shouldLogConsumerRecordMessage() {
 		RetryTopicConfigurer.LoggingDltListenerHandlerMethod method =
 				new RetryTopicConfigurer.LoggingDltListenerHandlerMethod();
 		method.logMessage(consumerRecordMessage);
-		then(consumerRecordMessage).should(never()).topic();
+		LogAccessor logger = (LogAccessor) ReflectionTestUtils
+				.getField(RetryTopicConfigurer.class, "LOGGER");
+		Assert.notNull(logger, "No LOGGER found in class " + RetryTopicConfigurer.class.getSimpleName());
+		if (logger.isInfoEnabled()) {
```

Comment:
Same `@LogLevels` trick?

Reply:
That's a really neat and useful annotation. Have you ever thought of proposing that for the JUnit project?

Reply:
😄  no, but I agree. We have it all over our with Gary projects, so it indeed could be a candidate for `spring-test`common module.